### DISCUSSION TITLE
Offer Torrent Downloads

### DISCUSSION THREAD

**Discussion by trymeouteh** [Kron4ek/Wine-Builds#72] [Ideas]
Consider offering torrent downloads of your builds by sharing the .torrent file and hosting the torrent on a personal computer or seedbox (A server for seeding torrents).

This way others can download and seed these Wine builds to help reduce the bandwidth on your end and to help distribute the wine builds globally.

**Comment by Kron4ek** [comment]
I don't think torrent is needed in this case. Wine builds are pretty small and GitHub usually gives decent download speeds.
